src/poliastro/czml/utils.py

```diff
@@ -109,9 +113,11 @@ def project_point_on_ellipsoid(x, y, z, a, b, c):
 
     Parameters
     ----------
-    x, y, z: Cartesian coordinates of point
+    coordinates x, y, z
```

Comment:
Rather than `coordinates`, shouldn't we specify they're `float`?

Reply:
sure, i will do it